docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+# This is a docker-compose file for use by Resource Providers
+services:
+  ipfs:
+    image: ipfs/kubo:v0.30.0
+    container_name: ipfs
+    restart: unless-stopped
+    ports:
+      - 4001:5001
```

Comment:
It looks like you have both 4001:5001 and 127.0.0.1:5001:5001 configured. To prevent any potential conflicts, you might want to consider updating these settings

Reply:
typo! good catch 🙏